src/CommunityToolkit.Maui.CameraView/CameraView.shared.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.ComponentModel;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Primitives;
+
+namespace CommunityToolkit.Maui.Views;
+
+public partial class CameraView : View, ICameraView
+{
+    static readonly WeakEventManager weakEventManager = new();
+
+    public static readonly BindableProperty CameraFlashModeProperty =
+        BindableProperty.Create(nameof(CameraFlashMode), typeof(CameraFlashMode), typeof(CameraView), CameraFlashMode.Off);
+
+    public static readonly BindableProperty IsTorchOnProperty =
+        BindableProperty.Create(nameof(IsTorchOn), typeof(bool), typeof(CameraView), false);
+
+    static readonly BindablePropertyKey isAvailablePropertyKey =
+        BindableProperty.CreateReadOnly(nameof(IsAvailable), typeof(bool), typeof(CameraView), false);
+
+    public static readonly BindableProperty IsAvailableProperty = isAvailablePropertyKey.BindableProperty;
+
+    static readonly BindablePropertyKey isCameraBusyPropertyKey =
+        BindableProperty.CreateReadOnly(nameof(IsCameraBusy), typeof(bool), typeof(CameraView), false);
+
+    public static readonly BindableProperty IsCameraBusyProperty = isCameraBusyPropertyKey.BindableProperty;
+
+    public static readonly BindableProperty? SelectedCameraProperty = BindableProperty.Create(nameof(SelectedCamera),
+        typeof(CameraInfo), typeof(CameraView), null, defaultBindingMode: BindingMode.TwoWay);
+
+    public static readonly BindableProperty ZoomFactorProperty =
+        BindableProperty.Create(nameof(ZoomFactor), typeof(float), typeof(CameraView), 1.0f, coerceValue: CoerceZoom, defaultBindingMode: BindingMode.TwoWay);
+
+    public static readonly BindableProperty CaptureResolutionProperty = BindableProperty.Create(nameof(CaptureResolution),
+        typeof(Size), typeof(CameraView), Size.Zero, defaultBindingMode: BindingMode.TwoWay);
+
+    public CameraFlashMode CameraFlashMode
+    {
+        get => (CameraFlashMode)GetValue(CameraFlashModeProperty);
+        set => SetValue(CameraFlashModeProperty, value);
+    }
+
+    public CameraInfo? SelectedCamera
+    {
+        get { return (CameraInfo?)GetValue(SelectedCameraProperty); }
```

Comment:
This can be an expression body to match the other properties.

```suggestion
        get => (CameraInfo?)GetValue(SelectedCameraProperty);
```